upload/library/SV/WarningImprovements/XenForo/Model/Warning.php

```diff
@@ -262,23 +262,24 @@ public function canViewWarningCategory(
         }
 
         $parentWarningCategoryId = $warningCategory['parent_warning_category_id'];
```

Comment:
Should use an empty() check to handle the case of the files have been updated, but the installer hasn't yet run to add the extra columns.